Grid-AdaptiveLayout/Grid-AdaptiveLayout/Pages/Counter.razor

```diff
@@ -1,18 +1,51 @@
 ﻿@page "/counter"
 
```

Comment:
add intro title in page as

Enable/Disable DataGrid Adaptive UI using Media Query Component